services/awp_azure_onboarding/awp_azure_onboarding.go

```diff
@@ -0,0 +1,209 @@
+package awp_azure_onboarding
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+const (
+	awpAzureGetOnboardingDataPath = "workload/agentless/azure/terraform"
+	azureOnboardingResourcePath   = "workload/agentless/azure/accounts"
+	cloudAccountsPath             = "AzureCloudAccount/"
+
+    ScanModeInAccountSub = "inAccountSub"
+    ScanModeInAccountHub = "inAccountHub"
+)
+
+type CreateAWPOnboardingDataRequest struct {
+	CentralizedId              string                    `url:"centralizedId"`
+}
+
+type AgentlessAzureTerraformOnboardingDataResponse struct {
+	Region                                     string `json:"region"`
+	AppClientId                                string `json:"appClientId"`
+	CloudAccountId                          string `json:"CloudAccountId"`
+	CentralizedCloudAccountId               string `json:"CentralizedCloudAccountId"`
+}
+
+type CloudAccountResponse struct {
+	ID                     string      `json:"id"`
+	Vendor                 string      `json:"vendor"`
+	Name                   string      `json:"name"`
+	ExternalAccountNumber  string      `json:"externalAccountNumber"`
+	Error                  interface{} `json:"error"`
+	IsFetchingSuspended    bool        `json:"isFetchingSuspended"`
+	CreationDate           string      `json:"creationDate"`
+	Credentials            Credentials `json:"credentials"`
+	IamSafe                interface{} `json:"iamSafe"`
+	NetSec                 NetSec      `json:"netSec"`
+	Magellan               bool        `json:"magellan"`
+	FullProtection         bool        `json:"fullProtection"`
+	AllowReadOnly          bool        `json:"allowReadOnly"`
+	OrganizationId         string      `json:"organizationId"`
+	OrganizationalUnitId   interface{} `json:"organizationalUnitId"`
+	OrganizationalUnitPath string      `json:"organizationalUnitPath"`
+	OrganizationalUnitName string      `json:"organizationalUnitName"`
+	LambdaScanner          bool        `json:"lambdaScanner"`
+	Serverless             Serverless  `json:"serverless"`
+	OnboardingMode         string      `json:"onboardingMode"`
+}
+
+type Credentials struct {
+	Apikey     interface{} `json:"apikey"`
+	Arn        string      `json:"arn"`
+	Secret     interface{} `json:"secret"`
+	IamUser    interface{} `json:"iamUser"`
+	Type       string      `json:"type"`
+	IsReadOnly bool        `json:"isReadOnly"`
+}
+
+type NetSec struct {
+	Regions []Region `json:"regions"`
+}
+
+type Region struct {
+	Region           string `json:"region"`
+	Name             string `json:"name"`
+	Hidden           bool   `json:"hidden"`
+	NewGroupBehavior string `json:"newGroupBehavior"`
+}
+
+type Serverless struct {
+	CodeAnalyzerEnabled           bool `json:"codeAnalyzerEnabled"`
+	CodeDependencyAnalyzerEnabled bool `json:"codeDependencyAnalyzerEnabled"`
+}
+
+type AgentlessAccountSettings struct {
```

Comment:
I think add "Azure" to the name "AgentlessAzureAccountSettings"
just to avoid exact name like AWS,

Reply:
done